.github/workflows/integration.yml

```diff
@@ -0,0 +1,123 @@
+name: integration
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v2
+    
+    - name: Set up repository
+      uses: actions/checkout@v3
+      with:
+          ref: integration
+    
+    - name: Install go
+      run: |
+        sudo make go-install
+        make prereq
+        source ~/.bashrc
+        
+    - name: Download scripts for generating certs
+      run: |
+        mkdir -p /tmp/certs
+        wget -P /tmp/certs https://raw.githubusercontent.com/ODIM-Project/ODIM/main/build/cert_generator/generate_client_crt.sh 
+        wget -P /tmp/certs https://raw.githubusercontent.com/ODIM-Project/ODIM/main/build/cert_generator/generate_etcd_certs.sh 
+        wget -P /tmp/certs https://raw.githubusercontent.com/ODIM-Project/ODIM/main/build/cert_generator/generate_odimra_cert.sh 
+
+    - name: Generate certs
+      run: |
+        sed -i 's/URP/device-manager/' /tmp/certs/generate_odimra_cert.sh
+        cp /tmp/certs/* build/certs
+        cd $GITHUB_WORKSPACE/build/certs
+        chmod +x generate_odimra_cert.sh 
+        ./generate_odimra_cert.sh deviceManager
+        
+    - name: Install Device Manager and ODIM's services
```

Comment:
There is a missing Device Manager config update step - password are being removed from source code, so you will have to add them here, as an additional step.